mono/mini/mini.h

```diff
@@ -1698,6 +1698,12 @@ extern const char MONO_ARCH_CPU_SPEC [];
 extern const guint16 MONO_ARCH_CPU_SPEC_IDX(MONO_ARCH_CPU_SPEC) [];
 #define ins_get_spec(op) ((const char*)&MONO_ARCH_CPU_SPEC + MONO_ARCH_CPU_SPEC_IDX(MONO_ARCH_CPU_SPEC)[(op) - OP_LOAD])
 
+static inline unsigned
+ins_get_size (int opcode)
```

Comment:
This can return simply an int.

Reply:
Ok. It has to cast to unsigned char first though, to get the full 0-255 range.